config/config.msft.public-cloud-overlay.yaml

```diff
@@ -0,0 +1,326 @@
+$schema: config.schema.json
+defaults:
+  # don't add any defaults here. only the clouds section is used for configuration
+  # global defaults can be found in https://github.com/Azure/ARO-HCP/blob/main/config/config.yaml
+clouds:
+  public:
+    defaults:
+      releaseApprover:
+        id: feca6a27-8f05-4abe-b9e4-e1185f5833ab
+        name: TM-AzureRedHatOpenShift-HCP-Leads
+      # Global
+      global:
+        subscription: hcp-global
+        rg: global-shared-resources
+        region: uksouth
+        globalMSIName: global-ev2-identity
+        secretsToSyncDir: 'msft-{{ .ctx.environment }}/arohcp{{ .ctx.environment }}-global'
+      # DNS
+      dns:
+        baseDnsZoneRG: global-shared-resources
+      # Monitoring
+      monitoring:
+        grafanaZoneRedundantMode: Disabled
+        grafanaMajorVersion: "11"
+      # Maestro
+      maestro:
+        certIssuer: OneCertV2-PrivateCA
+      # RP Frontend
+      frontend:
+        cert:
+          issuer: OneCertV2-PublicCA
+      # Cluster Service
+      clustersService:
+        azureOperatorsManagedIdentities:
+          # newly configured ARO-HCP builtin roles:
+          clusterApiAzure:
+            roleName: Azure Red Hat OpenShift Hosted Control Planes Cluster API Provider
+          controlPlane:
+            roleName: Azure Red Hat OpenShift Hosted Control Planes Control Plane Operator
+          # Classic roles:
+          cloudControllerManager:
+            roleName: Azure Red Hat OpenShift Cloud Controller Manager
+          ingress:
+            roleName: Azure Red Hat OpenShift Cluster Ingress Operator
+          diskCsiDriver:
+            roleName: Azure Red Hat OpenShift Disk Storage Operator
+          fileCsiDriver:
+            roleName: Azure Red Hat OpenShift File Storage Operator
+          imageRegistry:
+            roleName: Azure Red Hat OpenShift Image Registry Operator
+          cloudNetworkConfig:
+            roleName: Azure Red Hat OpenShift Network Operator
+          kms:
+            roleName: Key Vault Crypto User
+      # Logs
+      logs:
+        mdsd:
+          cert:
+            name: logs-mdsd
+            type: x-pem-file # GCS certificate file in PEM format
+            issuer: OneCertV2-PrivateCA
+      # SVC cluster settings
+      svc:
+        subscription: "hcp-{{ .ctx.environment }}-svc-{{ .ctx.region }}"
```

Comment:
Good candidate for promotion up to top-level default? Then let dev cloud override with the hard-coded name?

Reply:
I'm thinking from the perspective of someone building out a sovereign cloud in MSFT env - ideally they don't need to copy this block (or any of the blocks in this overlay that are fairly generic but just-so-happen-to-be overridden by `dev` cloud.) So maybe the litmus test is - if this naming scheme would also be the one used in Fairfax or Mooncake, let's see if we can hoist it up to the top-level defaults and just let `dev` override.